### PR TITLE
Unify 2nd generation models

### PR DESCRIPTION
**Pull request by Sgch**
This PR unifies the device tree for the 2nd generation models of the same hardware.

Affects models:
- PW-G4200: unified to PW-A7200
- PW-G5300: same as above

### CODE REVIEW

File: arch/arm/mach-imx/mxs/Kconfig [hunk 1]
```diff
@@ -78,15 +78,7 @@ config TARGET_PWSH7
...
-	bool "Support PW-A7200"
-	select BOARD_EARLY_INIT_F
-
-config TARGET_PWG4200
-	bool "Support PW-G4200"
-	select BOARD_EARLY_INIT_F
-
-config TARGET_PWG5300
-	bool "Support PW-G5300"
+	bool "Support PW-A7200 series"
```
Comment (puhitaku) [inline arch/arm/mach-imx/mxs/Kconfig:81]: Describing all compatible models will help other users.

```suggestion
	bool "Support PW-A7200 series"
	help
	  Compatible with: G4200, G5200, G5300, A7200, A7300, A7400, A9100, A9200, A9300
```

**Review by puhitaku — CHANGES_REQUESTED** [review]
I got the idea to have additional help text in Kconfig.
Functional changes all look OK.

**Comment by Sgch** [comment]
Sorry, regarding b11cab49a0f9ed72a043c660f63fe9cc10bff611 I missed the spelling error.

**Review by puhitaku — APPROVED** [review]
f6ca37ea-513b-474f-9339-f429e09d0c19

**Pull request merged**